tests/model/Megatron_GPT2/ds_gpt2_test.sh

```diff
@@ -91,9 +91,9 @@ gpt_options=" \
        ${ds_opt} \
        ${zero_opt} \
 "
-
+DEEPSPEED_PORT=29600
```

Comment:
The default master_port is 29500, just curious why we're changing it here? Not really a big deal, but just curious haha :)

Reply:
Yes, this was to run model tests concurrently with unit tests :).